webapp/settings.py

```diff
@@ -2,6 +2,9 @@
 Django project settings
 """
 
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
```

Comment:
The only need for `BASE_DIR` is so we can do things like `os.path.join(BASE_DIR, 'templates')`. However, in my experience we never actually need to use absolute paths for directories in these cases. If we just put `'templates'` and `'django-error.log'` directly, I think it would work. Actually, I'll test this out myself in a sec when I do the QA. In which case I'd rather not bother having this `BASE_DIR` setting.